package RuntimeException;

public class NumberFormatException {

	public static void main(String[] args) {
			
		String data1 = "100";
		String data2 = "a100";
		String str = null;

		
		int value1 = Integer.parseInt(data1);
		int value2 = 0;
		
//		try {
//			System.out.println(str.toString());
//		   	System.out.println(value1 / 0);
//			 value2 = Integer.parseInt(data2); // NumberFormatException 발생
//		}
//		catch(ArithmeticException e) {
//			System.out.println("정수를 0으로 나누시면 안됩니다.");
//		}
//		catch(Exception e) {
//			System.out.println("숫자 형태의 구성이 아닙니다.");
//			value2 = 200;
		}
//			catch(Exception e) {
//			System.out.println("예외가 발생 했습니다.");
//		}
//		finally {
//			System.out.println("예외처리에 대한 실행을 완료 합니다.");
//		}
		
		
		
//		int result = value1 + value2;
//		System.out.println(data1 + " + " + data2 + " = " + result);
//	}

}
